pkg/securitycontext/util.go

```diff
@@ -215,6 +215,7 @@ var (
 	// https://github.com/moby/moby/blob/master/oci/defaults.go#L116-L134
```

Comment:
Should this be updated to https://github.com/moby/moby/blob/b138373c6d3768b482f348aca56bc631cc77bdbb/oci/defaults.go#L105-L123.

Reply:
Let me know if you want me to update that in this PR

Reply:
I prefer to update using the permalink link.

Reply:
nit: we can include both - permalink AND master link. This way it's easy to check the current status AND find the code we referred to. It may be useful if there will be major renames in that repository.